data/en/test_dropdown_optional.json

```diff
@@ -0,0 +1,47 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Dropdown Mandatory",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "groups": [{
+        "id": "dropdown",
+        "title": "Dropdown Optional",
+        "blocks": [{
+                "type": "Question",
+                "id": "dropdown-optional",
+                "questions": [{
+                    "type": "General",
+                    "id": "dropdown-optional-question",
+                    "title": "Which football team do your support?",
+                    "answers": [{
+                        "type": "Dropdown",
+                        "id": "dropdown-optional-answer",
+                        "mandatory": false,
+                        "label": "Football team",
+                        "description": "Your favourite team from the Premier League.",
```

Comment:
We should remove description from at least one of the test surveys so we test a dropdown field without a description.

Reply:
@ajmaddaford Removed it from `test_dropdown_mandatory_with_overridden_error.json`